src/devices/device_models/common/duration_constraints.jl

```diff
@@ -126,30 +119,34 @@ function device_duration_look_ahead(ps_m::CanonicalModel,
     ps_m.constraints[name_down] = JuMPConstraintArray(undef, set_names, time_steps)
 
 
-    for t in time_steps[2:end], (ix,name) in enumerate(set_names)
+    for t in time_steps, (ix,name) in enumerate(set_names)
         # Minimum Up-time Constraint
         lhs_on = JuMP.GenericAffExpr{Float64, _variable_type(ps_m)}(0);
         for i in t-duration_data[ix].up:t
-            if in(time_steps,i)  
-                JuMP.add_to_expression!(lhs_on,1,ps_m.variables[var_names[1]][name,i])
+            if in(i,time_steps)  
```

Comment:
doing for `i in time_steps` will be easier to read.

Reply:
Sure, I'll make that change